packages/odyssey-storybook/src/guidelines/Roadmap/roadmapData.tsx

```diff
@@ -10,214 +10,102 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import { DataTableRowData, Status } from "@okta/odyssey-react-mui";
+import { DataTableRowData, Status, Tooltip } from "@okta/odyssey-react-mui";
 import { DataTableColumn } from "@okta/odyssey-react-mui";
 
+import rawData from "./roadmap.json";
+export const data: OdysseyComponent[] = rawData as OdysseyComponent[];
+
 export type OdysseyComponent = {
   name: string;
-  status: "Fully released" | "In Labs" | "In progress" | "Not started";
-  startDate: string;
-  labsRelease: string;
-  fullRelease: string;
+  type: string;
+  status: "" | "Released" | "In Labs" | "In progress" | "Not started";
+  define: string;
+  design: string;
+  develop: string;
+  deliverableTiming: string;
 };
 
 export const columns: DataTableColumn<DataTableRowData>[] = [
   {
     accessorKey: "name",
     header: "Name",
     enableHiding: false,
+    size: 325,
+  },
+  {
+    accessorKey: "type",
+    header: "Type",
+    enableHiding: true,
+    size: 200,
   },
   {
     accessorKey: "status",
     header: "Status",
-    Cell: ({ cell }) => {
-      const value = cell.getValue<string>();
+    size: 200,
+    Cell: ({ cell, row }) => {
+      const statusValue = cell.getValue<string>();
+      const defineValue = row.original.define;
+      const designValue = row.original.design;
+      const developValue = row.original.develop;
       const severity =
```

Comment:
@benlister-okta this ternary is pretty hard to read. Can we just keep these statuses in an object or map and reference them that way? LIke
```js
const severity = {
 "Released": "success"...
 ```